website_cookiefirst/views/portal_template.xml

```diff
@@ -1,10 +1,7 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <odoo>
-    <template id="footer_custom" inherit_id="website.footer_custom">
-        <xpath
-            expr="//footer//div/span[hasclass('o_footer_copyright_name')]"
-            position="after"
-        >
+    <template id="footer_custom" inherit_id="website.layout">
+        <xpath expr="//footer//span/.." position="after">
```

Comment:
When I put the `...` (3 dots), it was for not writing the rest of the expression, but I think you still need to use the `hasclass` part to select the exact element.

Reply:
ok I check I will come back

Reply:
i specify the xpath now